map/package.json

```
@@ -24,7 +24,7 @@
   },
   "scripts": {
     "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
-    "build": "react-scripts build",
+    "build": "SKIP_PREFLIGHT_CHECK=true react-scripts build",
```

What are we actually skipping here? Can we better yet set this flag on the CI vs setting it on the script itself?

---

```suggestion
    "build": "react-scripts build",
```

and in `build-deploy-map.yaml`

```
SKIP_PREFLIGHT_CHECK=true  yarn workspace map build
```